### PR TITLE
Lesson 14

### PR DESCRIPTION
**Pull request by Alarlar**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -26,10 +26,61 @@ const skillsSection = document.getElementById('Skills');
...
   const skillItem = document.createElement('li');
-
   skillItem.textContent = skill;
-
   skillsList.appendChild(skillItem);
-}
\ No newline at end of file
+}
+
+const messageForm = document.forms.leave_message;
```
Comment (brandaogabriel7) [inline js/index.js]: Nice approach to retrieving the form!

File: index.html [hunk 2]
```diff
@@ -44,6 +49,28 @@ <h2>Connect</h2>
             <li><a href="https://github.com/Alarlar">https://github.com/Alarlar</a></li>
         </ul>
         </section>
+        <section>
```
Comment (brandaogabriel7) [inline index.html]: You forgot to add the `h2` heading "Leave a Message" to this section.

**Review by brandaogabriel7 — APPROVED** [review]
You did great, Dinar!

You fulfilled almost all of the requirements for this assignment. You just forgot to add one heading to your "Leave a Message" section.

Also, your HTML file indentation has inconsistencies. Please run the format code command in VSCode to fix it.

File: index.html (hunk 2 above)
Comment (Alarlar) [reply index.html]: Got it. 
Added **h2** heading _"Leave a Message"_ to this section.

**Comment by Alarlar** [comment]
Gabriel, let me know when I can Merge pull request.

**Pull request merged**